### DISCUSSION TITLE
http sink bad request  (400) error

### DISCUSSION THREAD

**Discussion by LinTechSo** [vectordotdev/vector#15726] [Q&A] [answered]
Hi.
i set up vector[http]------>fluentd[http] and getting 
```
can't convert String into an exact number
```
error . 
using vector 0.26.0 version with json array request. 
is there any way to fix the vector http sink request?

**Comment by jszwedko** [comment, marked as answer]
Hi @LinTechSo ,

Can you share the Vector and FluentD configuration? If you are able to capture a request that would also be helpful.

**Reply by LinTechSo** [reply]
Hi @jszwedko . thank you for your reply. the problem have been fixed and it was my mistake in configuration . 
the vector works well as always :))
